chat-plugins/uno.js

```diff
@@ -0,0 +1,563 @@
+"use strict";
```

Comment:
``'use strict';``

tic marks seems to be the standard around PS, not quote marks